src/docs/development/data-and-backend/state-mgmt/options.md

```diff
@@ -35,6 +35,14 @@ A recommended approach.
 * [You might not need Redux: The Flutter edition][], by Ryan Edge
 * [Making sense of all those Flutter Providers][]
 
+## Riverpod
+
+A provider rewrite by the same author, compiled safe, more testable, more flexible
+
+* [Riverpod Homepage][]
+* [Riverpod Github][]
+* [Motivation for provider rewrite][]
```

Comment:
I think this is redundant because we get a good idea about the motivation from the homepage above. If you feel strongly about keeping it, please try to format the item similarly to other items on this page.